README.rst

```diff
@@ -43,6 +43,7 @@ Push the created index image to Pulp
     --bundle container-registry.example.com/bundle/image:123
     --arch x86_64
     --skip-quay
+    --deprecation-list ["container-registry.example.com/index/bundle-image:latest"]
```

Comment:
Should be something like ` --deprecation-list container-registry.example.com/index/bundle-image:latest, container-registry.example.com/repo/bundle-image:2`..The `[.. ]` won'r work with the command line. It will have to be a comma separated str

Reply:
good point